### PR TITLE
Optional multiple support

### PR DESCRIPTION
**Pull request by DimitarChristoff**
- on older iphones, multiple attribute was causing a lot of issues.
- additionally - rewrote some stuff for better ES2015 compat and amended the npm build script to use `--out-dir` as `-o dir/index.js` was causing ENOENT when `lib` did not exist.
- made events callbacks pass the instance as context for consistency/API.

### CODE REVIEW

**Comment by DimitarChristoff** [comment]
similar issues described here https://github.com/FineUploader/fine-uploader/issues/990 etc and links to chromium bugs relevant.

**Comment by DimitarChristoff** [comment]
done && merged upstream changes. I have kept the const/let and es2015 style imports as per my branch though it makes little difference. renamed `options` to `inputAttributes = {}`

**Comment by okonet** [comment]
This is looking awesome! I'm merging it now and will release a new version soon.

File: index.js [hunk 1]
```diff
@@ -1,5 +1,9 @@
-var React = require('react');
-var accept = require('attr-accept');
+import React from 'react';
+import accept from 'attr-accept';
+
+const supportMultiple = document && docuemnt.createElement ?
```
Comment (paramaggarwal) [inline index.js]: typo `docuemnt`

**Pull request merged**